calc.py

```diff
@@ -0,0 +1,82 @@
+# ! python 3
+# This program will take inputs one at a time from the user and do the operation
+# instructed in the order it was given then print out that function with the result.
+
+# Function that asks for input and tests if it's a number or not
+def getNums(message):
```

Comment:
Later on when you get more comfortable with the language you can check out a styleguide. This can help guide you on how to name things, like variables or functions. ex [here](https://www.python.org/dev/peps/pep-0008/#function-names)